locust/env.py

```diff
@@ -125,5 +125,5 @@ def create_web_ui(self, host="", port=8089, auth_credentials=None):
         :param port: Port that the web server should listen to
         :param auth_credentials: If provided (in format "username:password") basic auth will be enabled
```

Comment:
Could you add docstrings for the `tls_cert` and `tls_cert` params to the `create_web_ui()` method? (Since this method is part of the public API, and included in the API docs)

Reply:
Whoops yeah sure thing, missed this one.